### DISCUSSION TITLE
Next stable release

### DISCUSSION THREAD

**Discussion by ghost** [Doraku/DefaultEcs#182] [Q&A]
Hey!
I'd like to know if there are any plans for a stable release anytime soon? Specifically through nuget. I can't get any of the ci packages, and I'm sadly stuck to 0.17.2 :mfw:

**Comment by Doraku** [comment]
Hello! Sadly I don't want to make an official version yet as more breaking changes are coming (mainly splitting the serialization in its own package to remove some dependencies that can cause issue in unity). Would a pre-release version on nuget.org be ok for you? I created the tag `0.18.0-beta01` so it should be available soon.

**Reply by ghost** [reply]
It's perfectly ok! Thank you so much for the updated package :0
